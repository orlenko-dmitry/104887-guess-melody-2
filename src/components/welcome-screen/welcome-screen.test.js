import React from 'react';
import {render} from '@testing-library/react';

import WelcomeScreen from './welcome-screen.jsx';

it(`WelcomeScreen renders correctly`, () => {
  const {container} = render(
      <WelcomeScreen
        minutes={0}
        mistakesNumber={0}
        onStartGameClick={() => {}}
      />
  );

  expect(container.firstChild).toMatchSnapshot();
});
