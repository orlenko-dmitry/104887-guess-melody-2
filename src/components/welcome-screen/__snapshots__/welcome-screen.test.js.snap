// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeScreen renders correctly 1`] = `
<section
  class="welcome"
>
  <div
    class="welcome__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <button
    class="welcome__button"
    data-testid="start-game-btn"
  >
    <span
      class="visually-hidden"
    >
      Начать игру
    </span>
  </button>
  <h2
    class="welcome__rules-title"
  >
    Правила игры
  </h2>
  <p
    class="welcome__text"
  >
    Правила просты:
  </p>
  <ul
    class="welcome__rules-list"
  >
    <li>
      За 
      0
       минут нужно ответить на все вопросы.
    </li>
    <li>
      Можно допустить 
      0
       ошибки.
    </li>
  </ul>
  <p
    class="welcome__text"
  >
    Удачи!
  </p>
</section>
`;
